docs/src/pages/careers/lead-designer.md

```diff
@@ -0,0 +1,89 @@
+
+But while we are [the leading](https://tsh.io/state-of-frontend/#over-the-past-year-which-of-the-following-design-systems-was-your-favorite-go-to-solution) UI design system in the frontend space, the adoption of MUI is only at 25%. More importantly, the challenges of developers and designers to solve when creating UIs go way beyond the ones of design systems.
+We envision a future where MUI becomes the default toolkit for web developers to create UIs.
+It's why we started MUI Base, Joy UI, and MUI Toolpad. Design is key to achieving this goal.
+
+## About the role
+
+### What you'll do on a day-to-day basis
+
+Depending on the day, you'll:
+
+- Make the product experience more delightful, e.g. visually more appealing
+- Pay attention to all the smallest interface interactions so they yield an outstanding UX
+- Develop the brand
+- Refresh and expand the marketing pages, e.g. with a showcase & case study section
+- Design the next features across the different product
+- Shape the direction of the design kit products
+- **Solve organization issues**
+  - You will improve how the design responsibilities are covered.
+  - You will ensure the consistency of design and product direction throughout all the products and all customer-facing artifacts.
+- **Exercise team leadership**
+  - You will overseas the overall design quality
+  - You will manage designer's career
+  - You will hire and maintain a great team
+
+## About you
+
+We're looking for someone with an expert eye for visual and UX design. You have enough development skills (CSS-in-JS, JSX) to implement the designed interfaces. Details matter to you.
+
+### Skills you should have
+
+- **Expert-level knowledge of visual design**. You can create great-looking UIs. You spot small kerning issues without even paying attention which makes you care about the details more than most people.
+- **Expert-level knowledge of UX design**. You can build intuitive UIs, both at the scope of an application and of a component micro-interaction.
+- **Enough development skills to implement the designed interfaces**. Our React engineers are often busy implementing complex components. We are looking for somebody with enough CSS-in-JS and JSX skills to not depend on them to implement the design.
+- **Strong written and verbal communication skills**. As part of the team, you'll interface both directly and indirectly with community members and enterprise customers, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.
+
+### What would be nice if you had, but isn't required
+
+- **You've contributed to a design system before**. You have worked on a library of components in the past, maybe for internal usage in your company.
+- **You have other complementary skills**. We are a tiny team of 6 people. We could use any hidden skills that you might have. Maybe you are great at creating animations or creating illustrations.
```

Comment:
2 options — 
Small edit: update the number of our tiny team 
Larger question: Perhaps cut this line? I don't know if we are gaining much by explicitly adding a catch-all for detailed skills when we are more looking for managerial duties.

Reply:
Ah right, we are not 6 people anymore.